geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert2/TypeInference.scala

```diff
@@ -282,7 +282,7 @@ object TypeInference {
     * @return
     */
   private def binding(typed: ObjectType): String = typed match {
-    case STRING              => "String"
+    case STRING | null       => "String"
```

Comment:
String is the most 'general' type, right?

Reply:
yeah everything can be converted to a string